site/styles/_collections.scss

```diff
@@ -162,6 +162,9 @@
               }
             }
           }
+          .meta-item-synopsis {
```

Comment:
Would be good to use the exact classname `meta-item-synopsis hidden-md`?

Reply:
It would be nice but I don't want to risk breaking changes for clients with custom nonsense for such a minor change.